From 1896c111cf2856f5c8d511ca8ba52f5e98d8a6f4 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Mon, 22 Jul 2024 14:32:32 +0100
Subject: [PATCH 498/697] DT: bindings: add a dma-maxburst property to
 snps,designware-i2s

Do an end-run around ASoC in lieu of not being able to easily find the
associated DMA controller capabilities.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 .../devicetree/bindings/sound/snps,designware-i2s.yaml        | 4 ++++
 1 file changed, 4 insertions(+)

--- a/Documentation/devicetree/bindings/sound/snps,designware-i2s.yaml
+++ b/Documentation/devicetree/bindings/sound/snps,designware-i2s.yaml
@@ -69,6 +69,10 @@ properties:
       - description: RX DMA Channel
     minItems: 1
 
+  dma-maxburst:
+    description: FIFO DMA burst threshold limit
+    maxItems: 1
+
   dma-names:
     items:
       - const: tx
